dev/generate_bom.py

```diff
@@ -200,6 +200,19 @@ def write_docker_version_files(self):
       with open(config_file, 'w') as cfg:
         cfg.write(docker_tag)
 
+  def write_rpm_version_files(self):
+    """Write a file containing the version number for each microservice. Used
+    by build_release.py for rpm builds.
+    """
+    self.determine_and_tag_halyard()
+    for comp in dict(self.__component_versions.items() + self.__halyard_version.items()):
+      if comp == 'spinnaker':
```

Comment:
instead of duplicating this check, could you refactor it to something like
  if comp in EXCLUDE_VERSIONS_FOR_COMPONENTS
and define EXCLUDE_VERSIONS_FOR_COMPONENTS to ['spinnaker']?
(and update existing checks for this as well)